sqlglot/dialects/clickhouse.py

```diff
@@ -1393,3 +1412,20 @@ def values_sql(self, expression: exp.Values, values_as_table: bool = True) -> st
                 values_as_table = True
 
             return super().values_sql(expression, values_as_table=values_as_table)
+
+        def flatten_auto_refresh_prop(self, expression: exp.AutoRefreshProperty) -> str:
```

Comment:
Remove the entry from `TRANSFORMS` and rename this to `autorefreshproperty_sql`. Then it will be automatically associated with the corresponding expression.